### PR TITLE
update license notes to current state

### LINKED ISSUE #500: update third-party-license-notices

### PR DESCRIPTION
**Pull request by Gr33ndev**
Closes Gamify-IT/issues#500

### CODE REVIEW

File: LicenseNotes.json [hunk 1]
```diff
@@ -153,6 +153,61 @@
...
-  {"licTypeName": "Apache-2.0", "compName": "org.liquibase:liquibase-core", "compVersion": "4.19.0", "copyright": "Apache Commons Lang \n Copyright 2001-2021 The Apache Software Foundation \n This product includes software developed at \n The Apache Software Foundation (https://www.apache.org/).", "copyrightSource": "https://repo1.maven.org/maven2/org/liquibase/liquibase-core/4.19.0/liquibase-core-4.19.0.jar"}
-
+  {"licTypeName": "Apache-2.0", "compName": "org.liquibase:liquibase-core", "compVersion": "4.19.0", "copyright": "Apache Commons Lang \n Copyright 2001-2021 The Apache Software Foundation \n This product includes software developed at \n The Apache Software Foundation (https://www.apache.org/).", "copyrightSource": "https://repo1.maven.org/maven2/org/liquibase/liquibase-core/4.19.0/liquibase-core-4.19.0.jar"},
+
+  {"licTypeName": "MIT", "compName": "de.uni-stuttgart.gamify-it:authentification-validator", "compVersion": "v1.0.0", "copyright": "Copyright (c) 2022 GamifyIT – A Web-based Gaming Platform for Computer Science Education in the First Semesters", "copyrightSource": "https://github.com/Gamify-IT/authentification-validator/blob/v1.0.0/LICENSE"},
```
Comment (delvh) [inline LicenseNotes.json:158]: We document our own library? :grin:

File: LicenseNotes.json [hunk 2]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "Apache-2.0", "compName": "org.webjars:stomp-websocket ", "compVersion": "2.3.4", "copyright": "unknown", "copyrightSource": "https://github.com/jmesnil/stomp-websocket/blob/2.3.4/LICENSE.txt"},
+  {"licTypeName": "MIT", "compName": "net", "compVersion": "1.0.2", "copyright": "Copyright 2016 Sleepless Software Inc. All rights reserved.", "copyrightSource": "https://github.com/sleeplessinc/net/blob/master/LICENSE"},
+  {"licTypeName": "MIT", "compName": "sockjs", "compVersion": "0.3.24", "copyright": "Copyright (C) 2011 VMware, Inc.", "copyrightSource": "https://github.com/sockjs/sockjs-node/blob/v0.3.24/LICENSE"},
+  {"licTypeName": "MIT", "compName": "sockjs-client", "compVersion": "1.6.1", "copyright": "Copyright (c) 2011-2018 The sockjs-client Authors.", "copyrightSource": "https://github.com/sockjs/sockjs-client/blob/v1.6.1/LICENSE"},
+  {"licTypeName": "Apache-2.0", "compName": "stompjs ", "compVersion": "2.3.3", "copyright": "unknown", "copyrightSource": "https://github.com/jmesnil/stomp-websocket/blob/2.3.3/LICENSE.txt"},
+  {"licTypeName": "MIT", "compName": "@types/sockjs-client", "compVersion": "1.5.1", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "MIT", "compName": "@types/sockjs-client", "compVersion": "1.5.1", "copyright": "Copyrights are respective of each contributor listed at the beginning of each definition file.", "copyrightSource": "https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE"},
```

File: LicenseNotes.json [hunk 3]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "MIT", "compName": "@types/stompjs", "compVersion": "2.3.5", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "MIT", "compName": "@types/stompjs", "compVersion": "2.3.5", "copyright": "unknown", "copyrightSource": "https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE"},
```

File: LicenseNotes.json [hunk 4]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "Apache-2.0", "compName": "typescript", "compVersion": "3.9.10", "copyright": "Copyright (c) Microsoft Corporation. All rights reserved.", "copyrightSource": "https://github.com/microsoft/TypeScript/blob/v3.9.10/CopyrightNotice.txt"},
+  {"licTypeName": "Apache-2.0", "compName": "org.springframework.boot:spring-boot-starter-webflux", "compVersion": "2.7.3", "copyright": "Copyright (c) 2012-2022 Pivotal, Inc.", "copyrightSource": "https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-starter-webflux/2.7.3/spring-boot-starter-webflux-2.7.3.jar"},
+  {"licTypeName": "Apache-2.0", "compName": "org.hibernate.validator:hibernate-validator", "compVersion": "8.0.0.Final", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "Apache-2.0", "compName": "org.hibernate.validator:hibernate-validator", "compVersion": "8.0.0.Final", "copyright": "unknown", "copyrightSource": "https://github.com/hibernate/hibernate-validator/blob/8.0.0.Final/license.txt"},
```

File: LicenseNotes.json [hunk 5]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "BSD", "compName": "org.antlr:antlr4-runtime", "compVersion": "4.10.1", "copyright": "Copyright (c) 2012 Terence Parr and Sam Harwell", "copyrightSource": "https://www.antlr.org/license.html"},
+  {"licTypeName": "Apache-2.0", "compName": "com.github.tomakehurst:wiremock", "compVersion": "3.0.0-beta-4", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "Apache-2.0", "compName": "com.github.tomakehurst:wiremock", "compVersion": "3.0.0-beta-4", "copyright": "unknown", "copyrightSource": "https://github.com/wiremock/wiremock/blob/3.0.0-beta-4/LICENSE.txt"},
```

File: LicenseNotes.json [hunk 6]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "Apache-2.0", "compName": "javax.validation:validation-api", "compVersion": "2.0.1.Final", "copyright": "unknown", "copyrightSource": ""},
+  {"licTypeName": "MIT", "compName": "@popperjs/core", "compVersion": "2.11.16", "copyright": "Copyright (c) 2019 Federico Zivolo", "copyrightSource": "https://github.com/floating-ui/floating-ui/blob/v2.11.6/LICENSE.md"},
+  {"licTypeName": "MIT", "compName": "@rushstack/eslint-patch", "compVersion": "1.1.4", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "MIT", "compName": "@rushstack/eslint-patch", "compVersion": "1.1.4", "copyright": "Copyright (c) Microsoft Corporation. All rights reserved.", "copyrightSource": "https://github.com/microsoft/rushstack/blob/main/eslint/eslint-patch/LICENSE"},
```

File: LicenseNotes.json [hunk 7]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "MIT", "compName": "bootstrap", "compVersion": "5.2.1", "copyright": "Copyright (c) 2011-2022 Twitter, Inc.\nCopyright (c) 2011-2022 The Bootstrap Authors", "copyrightSource": "https://github.com/twbs/bootstrap/blob/v5.2.1/LICENSE"},
+  {"licTypeName": "MIT", "compName": "bootstrap-vue-3", "compVersion": "0.3.2", "copyright": "Copyright (c) 2021 BootstrapVue3", "copyrightSource": "https://github.com/bootstrap-vue/bootstrap-vue-next/blob/v0.3.2/LICENSE"},
+  {"licTypeName": "MIT", "compName": "@types/highlight.js", "compVersion": "10.1.0", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "MIT", "compName": "@types/highlight.js", "compVersion": "10.1.0", "copyright": "unknown", "copyrightSource": "https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE"},
```

File: LicenseNotes.json [hunk 8]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "MIT", "compName": "@types/marked ", "compVersion": "4.0.8", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "MIT", "compName": "@types/marked ", "compVersion": "4.0.8", "copyright": "unknown", "copyrightSource": "https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE"},
```

File: LicenseNotes.json [hunk 9]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "MIT", "compName": "@vue/test-utils", "compVersion": "2.2.7", "copyright": "Copyright (c) 2021-present vuejs", "copyrightSource": "https://github.com/vuejs/test-utils/blob/v2.2.7/LICENSE"},
+  {"licTypeName": "MIT", "compName": "vue-router", "compVersion": "4.1.6", "copyright": "Copyright (c) 2019-present Eduardo San Martin Morote", "copyrightSource": "https://github.com/vuejs/router/blob/v4.1.6/LICENSE"},
+  {"licTypeName": "BSD-3-Clause", "compName": "highlight.js", "compVersion": "11.7.0", "copyright": "Copyright (c) 2006, Ivan Sagalaev.", "copyrightSource": "https://github.com/highlightjs/highlight.js/blob/11.7.0/LICENSE"},
+  {"licTypeName": "Apache-2.0", "compName": "@typescript/lib-dom", "compVersion": "npm:@types/web@^0.0.93", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "Apache-2.0", "compName": "@typescript/lib-dom", "compVersion": "npm:@types/web@^0.0.93", "copyright": "unknown", "copyrightSource": "https://github.com/microsoft/TypeScript-DOM-lib-generator/blob/%40types/web%400.0.93/LICENSE.txt"},
```

File: LicenseNotes.json [hunk 10]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "BSD-2-Clause", "compName": "Sphinx", "compVersion": "4.5.0", "copyright": "Copyright (c) 2007-2022 by the Sphinx team", "copyrightSource": "https://github.com/sphinx-doc/sphinx/blob/v4.5.0/LICENSE"},
+  {"licTypeName": "MIT", "compName": "cachetools", "compVersion": "5.2.0", "copyright": "Copyright (c) 2014-2022 Thomas Kemmer", "copyrightSource": "https://github.com/tkem/cachetools/blob/v5.2.0/LICENSE"},
+  {"licTypeName": "MPL-2.0", "compName": "certifi", "compVersion": "2022.6.15", "copyright": "unknown", "copyrightSource": "https://github.com/certifi/python-certifi/blob/2022.06.15/LICENSE"},
+  {"licTypeName": "MIT", "compName": "charset-normalizer", "compVersion": "2.1.0", "copyright": "Copyright (c) 2019 TAHRI Ahmed R.", "copyrightSource": "https://github.com/Ousret/charset_normalizer/blob/2.1.0/LICENSE"},
+  {"licTypeName": "Apache-2.0", "compName": "google-api-core", "compVersion": "2.8.2", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "Apache-2.0", "compName": "google-api-core", "compVersion": "2.8.2", "copyright": "unknown", "copyrightSource": "https://github.com/googleapis/python-api-core/blob/v2.8.2/LICENSE"},
```

File: LicenseNotes.json [hunk 11]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "Apache-2.0", "compName": "google-api-python-client", "compVersion": "2.52.0", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "Apache-2.0", "compName": "google-api-python-client", "compVersion": "2.52.0", "copyright": "unknown", "copyrightSource": "https://github.com/googleapis/google-api-python-client/blob/v2.52.0/LICENSE"},
```

File: LicenseNotes.json [hunk 12]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "Apache-2.0", "compName": "google-auth", "compVersion": "2.9.0", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "Apache-2.0", "compName": "google-auth", "compVersion": "2.9.0", "copyright": "unknown", "copyrightSource": "https://github.com/googleapis/google-auth-library-python/blob/v2.9.0/LICENSE"},
```

File: LicenseNotes.json [hunk 13]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "Apache-2.0", "compName": "google-auth-httplib2", "compVersion": "0.1.0", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "Apache-2.0", "compName": "google-auth-httplib2", "compVersion": "0.1.0", "copyright": "unknown", "copyrightSource": "https://github.com/googleapis/google-auth-library-python-httplib2/blob/v0.1.0/LICENSE"},
```

File: LicenseNotes.json [hunk 14]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "Apache-2.0", "compName": "google-auth-oauthlib", "compVersion": "0.5.2", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "Apache-2.0", "compName": "google-auth-oauthlib", "compVersion": "0.5.2", "copyright": "unknown", "copyrightSource": "https://github.com/googleapis/google-auth-library-python-oauthlib/blob/v0.5.2/LICENSE"},
```

File: LicenseNotes.json [hunk 15]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "Apache-2.0", "compName": "googleapis-common-protos", "compVersion": "1.56.3", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "Apache-2.0", "compName": "googleapis-common-protos", "compVersion": "1.56.3", "copyright": "unknown", "copyrightSource": "https://github.com/googleapis/python-api-common-protos/blob/v1.56.3/LICENSE"},
```

File: LicenseNotes.json [hunk 16]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "BSD-3-Clause", "compName": "oauthlib", "compVersion": "3.2.0", "copyright": "Copyright (c) 2019 The OAuthlib Community", "copyrightSource": "https://github.com/oauthlib/oauthlib/blob/v3.2.0/LICENSE"},
+  {"licTypeName": "BSD-3-Clause", "compName": "protobuf", "compVersion": "4.21.6", "copyright": "unknown", "copyrightSource": ""},
+  {"licTypeName": "BSD-2-Clause", "compName": "pyasn1", "compVersion": "0.4.8", "copyright": "Copyright (c) 2005-2019, Ilya Etingof <etingof@gmail.com> All rights reserved.", "copyrightSource": "https://github.com/etingof/pyasn1/blob/v0.4.8/LICENSE.rst"},
+  {"licTypeName": "BSD-2-Clause", "compName": "pyasn1-modules", "compVersion": "0.2.8", "copyright": "Copyright (c) 2005-2019, Ilya Etingof <etingof@gmail.com>", "copyrightSource": "https://github.com/etingof/pyasn1-modules/blob/v0.2.8/LICENSE.txt"},
+  {"licTypeName": "MIT", "compName": "pyparsing", "compVersion": "3.0.9", "copyright": "unknown", "copyrightSource": "https://github.com/pyparsing/pyparsing/blob/pyparsing_3.0.9/LICENSE"},
+  {"licTypeName": "Apache-2.0", "compName": "requests", "compVersion": "2.28.1", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json]:
```suggestion
  {"licTypeName": "Apache-2.0", "compName": "requests", "compVersion": "2.28.1", "copyright": "unknown", "copyrightSource": "https://github.com/psf/requests/blob/v2.28.1/LICENSE"},
```

File: LicenseNotes.json [hunk 17]
```diff
@@ -153,6 +153,61 @@
...
+  {"licTypeName": "ISC", "compName": "requests-oauthlib", "compVersion": "1.3.1", "copyright": "Copyright (c) 2014 Kenneth Reitz.", "copyrightSource": "https://github.com/requests/requests-oauthlib/blob/v1.3.1/LICENSE"},
+  {"licTypeName": "Apache-2.0", "compName": "rsa", "compVersion": "4.8", "copyright": "Copyright 2011 Sybren A. Stüvel <sybren@stuvel.eu>", "copyrightSource": "https://github.com/sybrenstuvel/python-rsa/blob/version-4.8/LICENSE"},
+  {"licTypeName": "MIT", "compName": "six", "compVersion": "1.16.0", "copyright": "Copyright (c) 2010-2020 Benjamin Peterson", "copyrightSource": "https://github.com/benjaminp/six/blob/1.16.0/LICENSE"},
+  {"licTypeName": "Apache-2.0", "compName": "uritemplate", "compVersion": "4.1.1", "copyright": "unknown", "copyrightSource": ""},
```
Comment (delvh) [inline LicenseNotes.json:211]:
```suggestion
  {"licTypeName": "Apache-2.0", "compName": "uritemplate", "compVersion": "4.1.1", "copyright": "unknown", "copyrightSource": "https://github.com/python-hyper/uritemplate/blob/main/LICENSE.APACHE"},
```

**Review by delvh — COMMENTED** [review]
Ouch.
I can finally imagine your pain.

**Review by robojones — CHANGES_REQUESTED** [review]
Towercrush frontend now has two new dependencies with MIT license. Please add:

- https://www.npmjs.com/package/vue3-particles
- https://www.npmjs.com/package/tsparticles

**Review by delvh — APPROVED** [review]

**Pull request merged**